transformations/disability_transformation/test.json

```diff
@@ -0,0 +1,50 @@
+{
+  "type": "different_ability",
+  "test_cases": [
+    {
+      "class": "DifferentAbilityTransformation",
+      "inputs": {
+        "sentence": "He is blind."
+      },
+      "outputs": [{
+        "sentence": "He is person with a visual impairment "
+      }]
+    },
+    {
+      "class": "DifferentAbilityTransformation",
+      "inputs": {
+        "sentence": "He is so slow he's probably a retard."
+      },
+      "outputs": [{
+        "sentence": "He is so slow he's probably a person with a learning disablity "
+      }]
+    },
+    {
+      "class": "DifferentAbilityTransformation",
+      "inputs": {
+        "sentence": "John is deaf."
+      },
+      "outputs": [{
+        "sentence": "John is  person with a hearing disability "
```

Comment:
You might need to add some functionality to do cleanup: there is an extra space before and after the replaced phrase and the punctuation was removed.